pkgs/applications/networking/irc/communi/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchgit, libcommuni, qt5, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  name = "communi-${version}";
+  version = "ad1b9a30ed";
```

Comment:
Please change version to date. See https://nixos.org/nixpkgs/manual/#sec-package-naming